src/io/heap.zig

```diff
@@ -0,0 +1,379 @@
+const std = @import("std");
```

Comment:
you should link to where you got this from (https://github.com/mitchellh/libxev/blob/main/src/heap.zig)